src/Compilers/CSharp/Portable/BoundTree/NullabilityRewriter.cs

```diff
@@ -51,15 +51,13 @@ private BoundNode VisitBinaryOperatorBase(BoundBinaryOperatorBase binaryOperator
                 var right = (BoundExpression)Visit(currentBinary.Right);
                 var type = foundInfo ? infoAndType.Type : currentBinary.Type;
 
-#pragma warning disable IDE0055 // Fix formatting
                 // https://github.com/dotnet/roslyn/issues/35031: We'll need to update the symbols for the internal methods/operators used in the binary operators
                 currentBinary = currentBinary switch
-                    {
-                        BoundBinaryOperator binary => binary.Update(binary.OperatorKind, binary.ConstantValueOpt, binary.MethodOpt, binary.ResultKind, binary.OriginalUserDefinedOperatorsOpt, leftChild, right, type),
-                        BoundUserDefinedConditionalLogicalOperator logical => logical.Update(logical.OperatorKind, logical.LogicalOperator, logical.TrueOperator, logical.FalseOperator, logical.ResultKind, logical.OriginalUserDefinedOperatorsOpt, leftChild, right, type),
-                        _ => throw ExceptionUtilities.UnexpectedValue(currentBinary.Kind),
-                    };
-#pragma warning restore IDE0055 // Fix formatting
```

Comment:
@333fred it is not possible to suppress Format Document within a region of a document. The warning suppression here was ignored since the code inside it was automatically reformatted anyway.

Reply:
➡️ See discussion in https://github.com/dotnet/roslyn/pull/38562#discussion_r321929178